securing_apps/topics/saml/mod-auth-mellon.adoc

```diff
@@ -15,14 +15,17 @@ To configure mod_auth_mellon you'll need:
 ifeval::[{project_community}==true]
```

Comment:
It looks it has been this way since SSO 7.1: https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.1/html-single/securing_applications_and_services_guide/index#mod_auth_mellon

I assume it is correct or it is not used by many customers or they would file cases.

Reply:
Okay, makes sense. Thank you